spec/factories/phone_numbers.rb

```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :phone_number do
+    number "MyString"
+    type ""
+    contact nil
```

Comment:
You can update this to bind to the model, like so

> after(:build) do |phone_number|
>    phone_number.contact = create(:contact)
> end

in the place of `contact nil` and same in the other places.

Reply:
Oh thanks. Done